eclipse-formatter.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<profiles version="1">
+<profile kind="CodeFormatterProfile" name="ApacheKafka" version="1">
+    <!-- Import ordering -->
```

Comment:
Would it be possible to have only the import ordering in the formatter file and adding the other formatting rules incrementally?

Reply:
Let me have a try. 👍